flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_AirportsMonitor.js

```diff
@@ -197,19 +202,11 @@ class CDUAirportsMonitor {
                 // force spaces to emulate 4 columns
                 mcdu._labelElements[0][2].innerHTML = "&nbsp;&nbsp;&nbsp;";
                 for (let i = 0; i < (this.user_ap ? 5 : 4); i++) {
-                    mcdu._lineElements[i][2].innerHTML = mcdu._lineElements[i][2].innerHTML.replace(/_/g, "&nbsp;");
+                    mcdu._lineElements[i][2].innerHTML = mcdu._lineElements[i][2].innerHTML.replace(/_/g, "&nbsp;"); // TODO wtf
```

Comment:
🤔

Reply:
Need to take the time to understand why this is trying to directly edit the MCDU's HTML after setting the content itself through the proper API above... but out of scope for this PR probably.